d.configuration.md

```diff
@@ -306,9 +306,33 @@ kubernetes.io > Documentation > Tasks > Configure Pods and Containers > [Assign
 <p>
 
 ```bash
+# Flag --requests and --limits has been deprecated will be removed in future
 kubectl run nginx --image=nginx --restart=Never --requests='cpu=100m,memory=256Mi' --limits='cpu=200m,memory=512Mi'
 ```
 
+or
```

Comment:
we usually avoid using direct YAML in this repo since the repo's purpose is to help with CKAD exams.

Reply:
got it, thanks for the information, will close the PR.